install/orange-v0.7.0.sql

```diff
@@ -0,0 +1,371 @@
+    (1,'1','{}','meta','2016-11-11 11:11:11');
+
+/*!40000 ALTER TABLE `redirect` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table rewrite
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `rewrite`;
+
+CREATE TABLE `rewrite` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `key` varchar(255) NOT NULL DEFAULT '',
+  `value` varchar(2000) NOT NULL DEFAULT '',
+  `type` varchar(11) DEFAULT '0',
+  `op_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unique_key` (`key`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+LOCK TABLES `rewrite` WRITE;
+/*!40000 ALTER TABLE `rewrite` DISABLE KEYS */;
+
+INSERT INTO `rewrite` (`id`, `key`, `value`, `type`, `op_time`)
+VALUES
+    (1,'1','{}','meta','2016-11-11 11:11:11');
+
+/*!40000 ALTER TABLE `rewrite` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table waf
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `waf`;
+
+CREATE TABLE `waf` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `key` varchar(255) NOT NULL DEFAULT '',
+  `value` varchar(2000) NOT NULL DEFAULT '',
+  `type` varchar(11) DEFAULT '0',
+  `op_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unique_key` (`key`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+LOCK TABLES `waf` WRITE;
+/*!40000 ALTER TABLE `waf` DISABLE KEYS */;
+
+INSERT INTO `waf` (`id`, `key`, `value`, `type`, `op_time`)
+VALUES
+    (1,'1','{}','meta','2016-11-11 11:11:11');
+
+/*!40000 ALTER TABLE `waf` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table balancer
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `balancer`;
+
+CREATE TABLE `balancer` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `key` varchar(255) NOT NULL DEFAULT '',
+  `value` varchar(2000) NOT NULL DEFAULT '',
+  `type` varchar(11) DEFAULT '0',
+  `op_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unique_key` (`key`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+LOCK TABLES `balancer` WRITE;
+/*!40000 ALTER TABLE `balancer` DISABLE KEYS */;
+
+INSERT INTO `balancer` (`id`, `key`, `value`, `type`, `op_time`)
+VALUES
+    (1,'1','{}','meta','2016-11-11 11:11:11');
+
+/*!40000 ALTER TABLE `balancer` ENABLE KEYS */;
+UNLOCK TABLES;
+
+# Dump of table node
+# ------------------------------------------------------------
+
+CREATE TABLE `node` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT '',
+  `ip` varchar(20) NOT NULL DEFAULT '',
+  `port` smallint(6) DEFAULT '7777',
+  `api_username` varchar(50) DEFAULT '',
+  `api_password` varchar(50) DEFAULT '',
+  `op_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unique_key` (`ip`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
@sumory I thought a log table for the sync triggered user, time, return status on each node should be considered.

Reply:
good idea